spec/controllers/submissions_controller_spec.rb

```diff
@@ -121,6 +121,66 @@
       end
     end
 
+    context 'submitting a url' do
```

Comment:
A lot of the copied over expect statements I considered more as setup to ensure everything is configured properly rather than the actual test for my changes. I see what you mean though and will change it.